### PR TITLE
[#4083] Update Leave/Holiday page

### PR DESCRIPTION
**Pull request by lgheorghiu**
Scope
=====

The Leave page was updated to reflect the discussions we had during the team building regarding the number of leave days.

Changes
=======

All team members have the same fixed number of leave days to start with.
Based on current leave sheet, there are a number of 11 bank holidays for each listed country (exception UK) and 21 legal days.
The styleguide page was updated to reflect the total.

Also, open for discussions the *reward leave days* - each team member who has been for more than 2 years within the team gets 1 day +.

How to try and test the changes
===============================

reviewers: @adiroiban @hcs0 @brunogola @dumol @tim441 

Please let me know if you **agree** with the changes, if not please share your opinion.

### CODE REVIEW

**Comment by lgheorghiu** [comment]
needs-review

File: content/leave.rst [hunk 1]
```diff
@@ -39,8 +41,8 @@ Vacation
   operations are not disrupted.
 
 * When you run out of paid leave days, just continue to record them.
-  It is **OK** if you need more holiday days. Extra holiday days are not payed.
-  Not paid leave days will be subtracted from your salary
+  It is **OK** if you need more holiday days. Extra holiday days are not paid.
+  Not paid leave days will be substracted from your salary
```
Comment (hcs0) [inline content/leave.rst]: ``Not paid`` -> ``Unpaid``
``salary`` -> ``salary.``

Comment (adiroiban) [inline content/leave.rst]: Is `substracted` correct ? http://www.dictionary.com/browse/subtract

Comment (lgheorghiu) [inline content/leave.rst]: No. I'll revert.

File: content/leave.rst [hunk 2]
```diff
@@ -6,7 +6,9 @@ Leave / Holiday
 General
 =======
 
-* We have 30 days of paid holidays per year (21 legal leave + 9 bank holidays)
+* We have 32 days of paid holidays per year (21 legal leave + 11 bank holidays)
+
+* After 2 years within the team, another leave day is added to the total amount.
```
Comment (dumol) [inline content/leave.rst]: Is this after every two years? Or just after the first two years? Please make it as clear as possible. Thanks!

Comment (adiroiban) [inline content/leave.rst]: It should be after every year.

**Pull request merged**